discussions/api/v1/urls.py

```diff
@@ -9,4 +9,9 @@
         views.UserDetailView.as_view(),
         name='retrieve_user'
     ),
+    url(
+        r'^subscriptions/(?P<_id>\w+)$', # TODO Look up our REST conventions
```

Comment:
Where are you seeing this route in the ruby app? The only routes I see are in `notifications_and_subscriptions.rb` and they are namespaced to /user/, e.g. `"#{APIPREFIX}/users/:user_id/subscriptions"`